### PR TITLE
[#6861] Platform: Added a cancel button in the Create New-enc.-at-rest

### PR DESCRIPTION
**Pull request by gaurav061**
…At-Rest Config UI

### CODE REVIEW

**Comment by gaurav061** [comment]
Added a cancel button to go back to list config page from create config page, Kindly add your comments

File: managed/ui/src/components/config/Security/KeyManagementConfiguration.js [hunk 1]
```diff
@@ -331,6 +338,12 @@ class KeyManagementConfiguration extends Component {
                   disabled={false}
                   btnType="submit"
                 />
+                <YBButton
+                  btnText={'Cancel'}
+                  btnClass={'btn btn-orange'}
+                  disabled={false}
+                  onClick={this.showListView}
+                />
```
Comment (sshev) [inline managed/ui/src/components/config/Security/KeyManagementConfiguration.js:346]: - no need to wrap static text into `{}` as it's not an expression
- boolean attribute with false value could be simply omitted
```jsx
<YBButton
  btnText="Cancel"
  btnClass="btn btn-orange"
  onClick={this.showListView}
 />
```

**Review by sshev — APPROVED** [review]
minor formatting note

File: managed/ui/src/components/config/Security/KeyManagementConfiguration.js (hunk 1 above)
Comment (gaurav061) [reply managed/ui/src/components/config/Security/KeyManagementConfiguration.js:346]: Done

**Review by sshev — APPROVED** [review]

**Pull request merged**